modin/experimental/engines/omnisci_on_native/test/test_dataframe.py

```diff
@@ -768,7 +768,7 @@ def groupby_count(df, cols, as_index, **kwargs):
         run_and_compare(groupby_count, data=self.data, cols=cols, as_index=as_index)
 
     @pytest.mark.xfail(
-        reason="Currently mean() passes a lambda into backend which cannot be executed on omnisci backend"
+        reason="Currently mean() passes a lambda into backend which cannot be executed on omnisci engine"
```

Comment:
Why engine?

Reply:
This should be fixed. I suggest performing all necessary changes in the code in this PR and changes in the docs as part of separate issue.

Reply:
fixed